### DISCUSSION TITLE
It Joins meeting Perfectly But I dont know it will leave

### DISCUSSION THREAD

**Discussion by CoderZ90** [TobiasPankner/Teams-Auto-Joiner#139] [Q&A]
Hey i have tried the bot its amazing it joins meeting perfectly but i dont know if it would leave the meeting and join other my leave_threshold_number is true. 

i tried before but it didnt work so..

**Comment by CoderZ90** [comment]
And also i dont want to it to keep checking meeting again and again because of that i cannot focus on screen if i need to write the bot keeps coming back and rejoining and also it keeps checking participants list even if we enabled that pause thingy

**Reply by TobiasPankner** [reply]
First of all, leave_threshold_number should be a number and not true/false, please read the readme.  
Secondly, the bot still has to check the meeting members even if you have activated pause_search. This is so it can know when to leave.

**Reply by CoderZ90** [reply]
Thankyou TobiasPankner it worked you are a very pro python developer.

😊

**Reply by TobiasPankner** [reply]
Haha thanks
